ver/jp/asm/os/bzero.s

```diff
@@ -0,0 +1,59 @@
+.include "macro.inc"
```

Comment:
are any of these handwritten asm files (not in nonmatchings) the same as us? I think it might be worth having a version agnostic "asm" dir for situations like this and referring to it from both yamls